src/Ant/Checkbox.elm

```diff
@@ -159,3 +170,28 @@ toHtml checked (Checkbox config) =
             []
         , H.span [ Attr.class checkboxCustomCheckmarkClass ] []
         ]
+    -}
+
+    H.label 
+        [ Attr.class checkboxLabelClass
+        , Attr.class (checkedClass checked checkboxLabelClass)
+        , Attr.class (disabledClass config.disabled checkboxLabelClass) 
+        ]
+        [ H.span 
+            [ Attr.class checkboxClass
+            , Attr.class (checkedClass checked checkboxClass)
```

Comment:
I don't understand why the `checked` class needs to be applied to both the `label` and the `span`. Is this because you're trying to do some sort of CSS selector that requires both the label and the span to be "checked"? Please elaborate.

The way I would select the span with css when the checkbox is checked would be something like this:

```css
.elm-antd--checkbox-checked > span
```